lessons/module-2/testing-tdd-mocha-and-chai.md

```diff
@@ -66,57 +66,40 @@ When talking about what should be tested, we say that we want to test the **outc
 For example, let's say we have a quiz application that checks a user's answers and adds/removes points from their score:
 
 ```js
-class Question {
-  constructor(questionText, correctAnswer) {
-    this.questionText = questionText;
-    this.correctAnswer = correctAnswer;
-    this.score = 0;
+function createQuestion(questionText, correctAnswer) {
+  let question = {
+    questionText: questionText,
+    correctAnswer: correctAnswer
   }
+  return question
+}
 
-  checkAnswer(playerAnswer) {
-    if (playerAnswer === this.correctAnswer) {
-      this.score++
-    } else {
-      this.score--
-    }
-    return this.score
+checkAnswer(question, playerAnswer, score) {
+  if (playerAnswer === question.correctAnswer) {
+    this.score++
```

Comment:
I don't think we need `this` anymore for this function.

Reply:
good catch! fixed